resolwe_bio/processes/support_processors/spikeins_qc.yml

```diff
@@ -0,0 +1,68 @@
+- slug: spikein-qc
+  name: Spike-ins quality control
+  requirements:
+    expression-engine: jinja
+    resources:
+      cores: 1
+    executor:
+      docker:
+        image: resolwebio/rnaseq:4.2.0
+  data_name: '{{ sample|sample_name|default("?") }} spike-ins QC'
+  version: 0.0.1
+  type: data:spikeins
+  category: other
+  persistence: CACHED
+  description: >
+    Plot spike-ins measured abundances for sample`s quality control. The process will output
+    graphs showing the correlation between known concentration of ERCC spike-ins and sample`s
+    measured abundance.
+  input:
+    - name: samples
+      label: List of samples
```

Comment:
How about: "Expressions with spike-ins"? @jkokosar would you review the process and input descriptions?